docs/boards/Board - FLYWOOF7DUAL.md

```diff
@@ -0,0 +1,107 @@
+* VBAT/CURR/RSSI sensors input
+* Suppose IRC Tramp/smart audio/FPV Camera Control/FPORT/telemetry
+* Supports SBus, Spektrum1024/2048, PPM. No external inverters required (built-in).
+* Supports I2C device extend(baro/compass/OLED etc)(socket)
+* Supports GPS (socket)
+
+### All uarts have pad on board 
+| Value | Identifier   | RX   | TX   | Notes                                                                                       |
+| ----- | ------------ | -----| -----| ------------------------------------------------------------------------------------------- |
+| 1     | USART1       | PA10 |  PA9 | FOR SBUS IN(inverter build in)     
+| 2     | USART2       | PA3  |  PA2 | USE FOR TRAMP/smart audio                                                                   |
+| 3     | USART3       | PB11 |  PB10| USE FOR GPS                                                                                 |
+| 4     | USART4       | PA1  |  PA0 | PAD USE FOR TRAMP/smart audio                                                               |
+| 5     | USART5       | PD2  |  PC12| PAD ESC sensor                                                                              |
+| 6     | USART6       | PC7  |  PC6 | PAD                                                                                         |
+
+
+### I2C with GPS port together.Use for BARO or compass etc 
+| Value | Identifier   | function |  pin   | Notes                                                                                 |
+| ----- | ------------ | ---------| -------| ------------------------------------------------------------------------------------- |                                                                                      
+| 1     | I2C1         |    SDA   |  PB7   | with GPS outlet
+| 2     | I2C1         |    SCL   |  PB6   | with GPS outlet
+
+
+### Buzzer/LED output 
+| Value | Identifier   | function |  pin   | Notes                                                                                 |
+| ----- | ------------ | ---------| -------| ------------------------------------------------------------------------------------- |                                                                                      
+| 1     | LED0         |    LED   |  PC15  | 
+| 2     | BEEPER       |    BEE   |  PC14  | 
+
+
+### VBAT input with 1/10 divider ratio,Current signal input,Analog/digit RSSI input
+| Value | Identifier   | function  |  pin  | Notes                                                                                 |
+| ----- | ------------ | ----------| ------| ------------------------------------------------------------------------------------- |                                                                                       
+| 1     | ADC1         |    VBAT   |  PC1  |  DMA2_Stream0
+| 2     | ADC1         |    CURR   |  PC0  |  DMA2_Stream0
+| 3     | ADC1         |    RSSI   |  PC2  |  DMA2_Stream0
+
+
+### 8 Outputs, 1 PPM input 
```

Comment:
there's only 6 outputs, not 8.

Reply:
I have modified it. Thank you for your reminding

Reply:
@hydra